core/templates/pages/contributor-dashboard-admin-page/services/contributor-dashboard-admin-backend-api.service.ts

```diff
@@ -121,11 +121,14 @@ export class ContributorDashboardAdminBackendApiService {
   async removeContributionReviewerAsync(
       username: string, category: string, languageCode: string): Promise<void> {
     var url = this.urlInterpolationService.interpolateUrl(
-      PageConstants.REMOVE_CONTRIBUTION_RIGHTS_HANDLER_URL, { category });
+      PageConstants.CONTRIBUTION_RIGHTS_HANDLER_URL, { category });
+    var params = {
+      username: username,
+      language_code: languageCode
+    };
     return new Promise((resolve, reject) => {
-      this.http.put<void>(url, {
-        username: username,
-        language_code: languageCode
+      this.http.delete<void>(url, {
+        params
```

Comment:
Why this change?

Reply:
This change us necessary because the remove is changed to a delete request according to this [suggestion](https://github.com/oppia/oppia/pull/12566#discussion_r618154679).